systemd/initial-setup.service

```diff
@@ -13,6 +13,7 @@ Type=oneshot
 TimeoutSec=0
 StandardInput=tty
 StandardOutput=tty
+LimitNOFILE=9001
```

Comment:
Oops - that's actually a debuging leftover and should not be needed - I'll remove it. :)

And why 9001 ? Well, [that's obvious](https://www.youtube.com/watch?v=SiMHTK15Pik). ;-)